nixos/modules/config/users-groups.nix

```diff
@@ -639,10 +689,52 @@ in {
       "/etc/profiles/per-user/$USER"
     ];
 
+    # systemd initrd
+    boot.initrd.systemd = lib.mkIf config.boot.initrd.systemd.enable {
+      contents = {
+        "/etc/passwd".text = ''
```

Comment:
when initrd ssh on, the output is
```
-sh-5.2# cat /etc/passwd
messagebus:x:4:4::/var/empty:
nobody:x:65534:65534::/var/empty:
root:x:0:0::/var/empty:
sshd:x:1:1::/var/empty:
systemd-network:x:152:152::/var/empty:
```
I think root home directory should be `/root`, and the root shell should be `${boot.initrd.network.ssh.shell}`

Reply:
@mlyxshi That would require having a means of configuring a user's shell and home directory in `boot.initrd.systemd.users`. Given that's a whole new feature, let's save that for a followup Pr.